Sources/StreamChat/StateLayer/Chat.swift

```diff
@@ -186,6 +186,40 @@ public final class Chat {
         return MemberState(member: member, cid: cid, database: databaseContainer)
     }
     
+    // MARK: - Member List
+    
+    /// Loads an array of members for the specified query.
+    ///
+    /// - Parameters:
+    ///   - filter: The filter to use for fetching channel members.
+    ///   - sort: An array of sorting parameters.
+    ///   - pagination: The pagination configuration which includes a limit and an offset or a cursor.
+    ///
+    /// - Throws: An error while communicating with the Stream API.
+    /// - Returns: An array of paginated channel members for the query.
+    public func loadMembers(with filter: Filter<MemberListFilterScope>, sort: [Sorting<ChannelMemberListSortingKey>] = [], pagination: Pagination) async throws -> [ChatChannelMember] {
```

Comment:
Other platforms use the term of `queryMembers` but all our other pagination methods use `load`. Something to discuss when reviewing the state layer as a whole.